buildlib/pr/main.yml

```diff
@@ -188,6 +188,13 @@ stages:
     jobs:
     - template: static_checks.yml
 
+  - stage: MAD_Tests
+    dependsOn: [Static_check]
+    jobs:
+    - template: mad_tests.yml
+      parameters:
+        HCA: "mlx5_0:1"
```

Comment:
why we need this param here? who will control it from main.yml scope? Do we usually manage hard coded parameters from main.yml?

Reply:
Moved the hardcoded var into the Bash script.